doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -202,7 +202,7 @@ nRF5340 Audio
 nRF Desktop
 -----------
 
-|no_changes_yet_note|
+* Updated MCUboot bootloader configurations for the relevant board targets to enable the :kconfig:option:`CONFIG_FPROTECT` Kconfig option that is used to protect the bootloader partition against memory corruption.
```

Comment:
please add a comment on the release notes PR https://github.com/nrfconnect/sdk-nrf/pull/18192, providing the link the commit that drops this release note entries. I shall add those in the release notes PR.